### PR TITLE
Add recipe for crisprme

### PR DESCRIPTION
**Pull request by samuelecancellieri**
Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by samuelecancellieri** [comment]
@BiocondaBot please add label

**Comment by dpryan79** [comment]
@bioconda-bot please fetch artifacts

File: recipes/crisprme/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  skip: True  # [osx or py<36]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - libgomp  # [linux]
+  host:
+    - make
+    - boost
+    - python
```
Comment (dpryan79) [inline recipes/crisprme/meta.yaml:22]: It looks like you just need a `run` section, without `build` or `host`. But then you should also specify `python >3.6` in the `run:` section.

File: recipes/crisprme/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  name: crisprme
+  version: {{ version }}
+
+source:
+  url: https://github.com/samuelecancellieri/CRISPRme/archive/v{{ version }}.tar.gz
+  sha256: 427c755b0c489ded70bb7ada4b347302eab31aa51439c36e8ccf6375db3d4884
+  
+build:
+  number: 0
+  skip: True  # [osx or py<36]
```
Comment (dpryan79) [inline recipes/crisprme/meta.yaml:13]: This appears to be pure python/shell script package, so why is OSX not supported?

**Review by dpryan79 — CHANGES_REQUESTED** [review]
A few questions.

File: recipes/crisprme/meta.yaml (hunk 2 above)
Comment (samuelecancellieri) [reply recipes/crisprme/meta.yaml:13]: crispritz package i use as core, is not supported by OSX, so installing this package is completely useless in OSX due to this incompatibility

File: recipes/crisprme/meta.yaml (hunk 1 above)
Comment (dpryan79) [reply recipes/crisprme/meta.yaml:22]: That'll be resolved by conda during installation, so there's no need for it here.

File: recipes/crisprme/meta.yaml (hunk 2 above)
Comment (dpryan79) [reply recipes/crisprme/meta.yaml:13]: OK, you can add `noarch: generic` then and remove the `skip:` line. This still won't be installable on OSX until crispritz supports OSX, so there's no issue there.

Comment (samuelecancellieri) [reply recipes/crisprme/meta.yaml]: done

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Comment by samuelecancellieri** [comment]
@bioconda-bot please merge

**Pull request merged**